pytket/extensions/cutensornet/tnstate/mps.py

```diff
@@ -358,6 +205,18 @@ def apply_gate(self, gate: Command) -> MPS:
 
         return self
 
+    def apply_scalar(self, scalar: complex) -> TNState:
+        """Multiplies the state by a complex number.
+
+        Args:
+            scalar: The complex number to be multiplied.
+
+        Returns:
+            ``self``, to allow for method chaining.
+        """
+        self.tensors[0] *= scalar

```

Comment:
Sorry for a dumb question - why only zeroth tensor? (is it because we have a product of them anyway?)

Reply:
Yeah, it's like in `z*(A*B) = (z*A)*B` and `z*(A⊗B) = (z*A)⊗B`.